src/model/company.profile.model.ts

```diff
@@ -0,0 +1,22 @@
+import { Address } from "model/data.types.model";
+export interface ICompanyDetails {
+    companyName?: string;
+    dateOfCreation?: string;
+    companyType?: string;
+    jurisdiction?: string;
+    companyNumber?: string;
+    registeredOfficeAddress?: Address;
+    // incorporationCountry?: string;
+    // principalAddress?: Address;
+    // isServiceAddressSameAsPrincipalAddress?: yesNoResponse;
+    // serviceAddress?: Address;
+    // email?: string;
+    // legalForm?: string;
+    // lawGoverned?: string;
+    // publicRegisterName?: string;
+    // publicRegisterJurisdiction?: string;
+    // registrationNumber?: string;
+    // isOnRegisterInCountryFormedIn?: yesNoResponse;
+}
+
```

Comment:
It would be good to clean up by deleting this file so that the inconsistencies can be easily resolved as we are using the CompanyProfile from node-sdk-api